### DISCUSSION TITLE
代码copy还是有一点异常

### DISCUSSION THREAD

**Discussion by koldle** [tangly1024/NotionNext#222] [Q&A] [answered]
#165 
描述: 该bug依然存在于当前版本中, 只会在最开始打开网页时得到一次触发的机会, 刷新页面后不会再触发  
复现方式: 多开几次无痕模式, 不是每次都会出现, 但基本每三四次就能触发

**Comment by tangly1024** [comment, marked as answer]
已重新修复该问题：
- 相关Issues
https://github.com/tangly1024/NotionNext/issues/225
- 相关分支合并
- https://github.com/tangly1024/NotionNext/pull/227

**Comment by vinson-lee01** [comment]
相关Issues
https://github.com/tangly1024/NotionNext/issues/225
相关分支合并
https://github.com/tangly1024/NotionNext/pull/227
